package.json

```diff
@@ -38,7 +38,7 @@
     "eslint-plugin-markdown": "^5.1.0",
     "eslint-plugin-n": "^17.16.2",
     "eslint-plugin-prettier": "^5.2.3",
-    "eslint-plugin-svelte": "^2.46.1",
+    "eslint-plugin-svelte": "^3.0.3",
```

Comment:
Most ESLint-related dependencies are likely installed via `@sveltejs/eslint-config`, so I don’t think we need to add them in this repository. If necessary, I will address it in a separate PR.